src/main/java/seedu/address/model/transaction/Transaction.java

```diff
@@ -50,6 +50,8 @@ public Transaction(Person payer, Amount amount, Description description, Date da
         this.amount = amount;
         this.description = description;
         this.payees = payees;
+
+        updatePayerAndPayeesBalance(amount, payer, payees);
     }
```

Comment:
Is this the line of code that caused the person panel list not updated after the first transaction?